requirements.txt

```diff
@@ -1,22 +1,33 @@
+# NOTE: Please try and keep this file tidy.
```

Comment:
Read this note :tada: 

Also: The list of packages below is the same as before. The only change is:

1. Every dependency found on prod has the current prod version pinned down.
2. They are grouped into some (crude) sections.
3. At the end are some packages that I simply didn't find on prod. It's possible that they were manually uninstalled. I'll have to follow up with @helgihg on those. Likely, we can remove them.

Point no. 3 is something we should figure out at some point.. But it's no big problem, since this file is no longer the actual dependencies that we should be installing by. That's what the `.freeze` files are for.